docs/proposals/2018-02-21-Distributor.md

```diff
@@ -0,0 +1,99 @@
+# Problem
+
+To date, ANS has represented the external origin of content through two main traits: `owner` and `source`.  The `source` trait covers the technical point of ingestion or creation (CMS, external id, etc) and is (mostly) straightforward. The `owner` trait is more convoluted, but is most reliably used to store the technical name of a content repository owner (organization id).
+
+However, there is also a need for data about the *legal* owner of a piece of content, including distribution rights and a general answer to the question of *who* entered a piece of content into the CMS. (This is the distinct from the question of *who authored the content* (`credits`) and *how the content was added* (`source`). Previously, this data has been haphazardly spread across `owner`, `source` and `additional_properties`. This has lead to confusion for users and conflicts for applications.
+
+# Proposal
+
+We propose that a new trait `distributor` be added to accomodate this use case and propose an implementation plan to phase out the previous locations of this data.
+
+For backwards compatibility, we suggest *no changes* to the existing fields:
+  * `owner.id`
+  * `owner.sponsored`
+  * `source.source_id`
+  * `source.system`
+  * `source.edit_url`
+  * `copyright`
+
+The following fields are to be *deprecated* and ultimately fall into desuetude:
+  * `source.name`
+  * `source.type`
+  * `owner.name`
+
+The following fields should be added:
+  * `distributor.name` -- The human-readable name of the person or organization controls the distribution rights for a piece of content. E.g. Reuters, AP, Getty
```

Comment:
Does this means "AP" and "Associated Press" will be treated as two different distributor?

Reply:
Yes. Do you think we should add an `id`?

Reply:
I'm not sure. If this field is to categorize a document then maybe? But we can leave it up to the users to maintain the uniqueness of the value.